### PR TITLE
feat: structure align output into www and Offline-HTML dirs

### PR DESCRIPTION
**Pull request by joanise**
### PR Goal? 

As agreed Wednesday, structure the `readalongs align` output like the web app's Web Bundle, with `www` and `Offline-HTML` subfolders.

### Feedback sought? 

run align with `-o html` and confirm you like the results

### Priority? 

normal

### Tests added? 

updated existing tests

### How to test? 

run `readalongs align` with various output options.

### Confidence? 

medium-high

### CODE REVIEW

**Comment by joanise** [comment]
Doh, I only fixed some tests. Turning this back into a draft at the moment.

**Comment by joanise** [comment]
OK, fixed, tests should pass now, this PR is ready to review.

File: docs/outputs.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Below is an example of a minimal implementation in a basic standalone html page.
 The above assumes the following structure, which is the default provided in the Web Bundle generated from the web app:
 
 ```txt
-web
+www
```
Comment (deltork) [inline docs/outputs.md:50]: We need to mention the readme.txt, it has WordPress deployment information.

File: test/test_align_cli.py [hunk 2]
```diff
@@ -187,11 +187,11 @@ def test_align_with_package(self):
         # print(results_html.output)
         self.assertEqual(results_html.exit_code, 0)
         self.assertTrue(
-            exists(join(output, "html.html")),
-            "succesful html alignment should have created html/html.html",
+            exists(join(output, "Offline-HTML", "html.html")),
+            "succesful html alignment should have created html/Offline-HTML/html.html",
```
Comment (deltork) [inline test/test_align_cli.py:191]: successful

File: readalongs/align.py [hunk 3]
```diff
@@ -950,41 +960,44 @@ def save_readalong(
...
 
     audio_path = save_audio(
-        audiofile=audiofile, output_base=output_base, audiosegment=audiosegment
+        audiofile=audiofile, output_base=bundle_base, audiosegment=audiosegment
     )
 
     if "html" in output_formats:
-        html_out_path = output_base + ".html"
+        offline_html_dir = os.path.join(output_dir, "Offline-HTML")
+        html_out_path = os.path.join(offline_html_dir, output_basename + ".html")
```
Comment (deltork) [inline readalongs/align.py:968]: I think the Offline file name should be index.html. `Offile-HTML\index.html`

**Review by deltork — CHANGES_REQUESTED** [review]
It all looks good!

File: readalongs/align.py (hunk 3 above)
Comment (joanise) [reply readalongs/align.py:968]: The title is harder to set in the CLI, and will frequently just have the default value, so I'd rather use the `output_basename`. Unless we check that it's not the default title, but let's make a separate issue about this, it's work I don't want to roll into this PR.

Comment (joanise) [reply readalongs/align.py:968]: Added #234 for this.

File: docs/outputs.md (hunk 1 above)
Comment (joanise) [reply docs/outputs.md:50]: Good point, thanks for catching this.

Comment (joanise) [reply docs/outputs.md:50]: It's mentioned higher up in the page, in the WordPress section.

**Comment by joanise** [comment]
Improvements pushed, this should be ready to re-review now. Thanks for the review!

**Review by deltork — APPROVED** [review]
Looks ready

**Pull request merged**